abdulfatai360/banka
# #165048919 Setup Project Development Environment

#### What does this PR do?
Setup the project development environment

#### Description of Task to be completed?
- Create a `package.json` file
- Have the server-side framework installed
- Have the transpiling tool and its essential plugins installed

#### How should this be manually tested?
- Run `git clone https://github.com/abdulfatai360/banka.git` in terminal to clone this repo onto your local machine
- Navigate to the repo by running `cd banka/` on terminal
- Ensure your working directory is clean by running  `git status`  command on terminal
- Run `npm install` to install project's dependencies
- Get a copy of the PR by typing  `git fetch origin pull/15/head:ch-setup-dev-env-165048919`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-setup-dev-env-165048919`
- Your directory will now be an exact copy of the PR.

#### What are the relevant pivotal tracker stories?
#165048919
